atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/OkEvent.java

```diff
@@ -32,15 +33,35 @@
 public abstract class OkEvent implements Event {
     public static final String TYPE = "ok";
 
+    public static final String UNLOCK_SUCCESS = "true";
+    public static final String UNLOCK_FAILURE = "false";
+    public static final String SUCCESS = "Dummy tokens";
+    public static final String FAILURE = "";
+
+    @Override
     public abstract long time();
 
     public abstract int process();
 
     @Nullable
     public abstract String value();
 
+    @JsonProperty("f")
+    public abstract String function();
+
     @Override
     public void accept(EventVisitor visitor) {
         visitor.visit(this);
     }
+
+    public boolean isFailure() {
+        return value() == null
+                || (function().equals(RequestType.UNLOCK) && value().equals(UNLOCK_FAILURE))
+                || value().equals(FAILURE);
+    }
+
+    public boolean isSuccessful() {
+        return value().equals(SUCCESS)
+                || (function().equals(RequestType.UNLOCK) && value().equals(UNLOCK_SUCCESS));
```

Comment:
Looking at this the two aren't generally complementary (e.g. an unlock which has some other value?), but would it be enough for our checkers to assume !failure is success?